src/main/java/com/microsoft/cse/helium/app/dao/ActorsDao.java

```diff
@@ -25,11 +28,28 @@
     final String _actorOrderBy = " order by m.name ";
     final String _actorOffset = " offset %d limit %d ";
 
+    final String _actorSelectById = _actorSelect + " and m.actorId = ";
+
+
     @Autowired
     public ActorsDao(IConfigurationService configService){
         super(configService);
     }
 
+    public Mono<Actor> getActorById(String _actorId) {
+
+        ObjectMapper objMapper = ObjectMapperFactory.getObjectMapper();
+
+        String readItemQuery = _actorSelectById + _actorId;
+        System.out.println("readItemQuery " + readItemQuery);
+
+       /* Mono<List<FeedResponse<CosmosItemProperties>>> feedResponse = this._context.getBean(CosmosClient.class).getDatabase(this._cosmosDatabase)
+                .getContainer(this._cosmosContainer).queryItems(readItemQuery, this._feedOptions).collectList(); */
+
+       return null;
```

Comment:
Is this intentional? Return null instead of an actual query? Also, in the commented code, it is List. Don't use List, it is a sync data type. Use Flux instead.

Reply:
This came in from pending PR, Since this is single item read, it will be Mono.